sw/otbn/crypto/BUILD

```diff
@@ -315,3 +315,21 @@ otbn_binary(
         ":rsa_verify_3072_rr",
     ],
 )
+
+otbn_library(
+    name = "x25519",
+    srcs = [
+        "x25519.s",
+    ],
+)
+
+otbn_binary(
```

Comment:
Are we planning to integrate these test snippets into our test infrastructure?

Reply:
SG. We can make this change in a follow up PR. Thanks for filling the issue to track this!